.github/ci-scripts/bash/build.sh

```diff
@@ -0,0 +1,1352 @@
+  aws configure set region ap-south-1
+}
+
+ awsConfigureTessellTools() {
+  aws configure set aws_access_key_id ${PROD_ASSET_ACCESS_KEY}
+  aws configure set aws_secret_access_key ${PROD_ASSET_SECRET_KEY}
+  aws configure set region ap-south-1
+}
+
+checkFlyway() {
+  if [ -d "flyway" ]; then
+    echo "The 'flyway' directory exists in this repository."
+    DOCKER_FLYWAY_IMAGE_NAME=$(echo ${GITHUB_REPOSITORY%-*} | cut -f2 -d'/')-flyway-migration
+    docker tag ${DOCKERHUB_ORG}/$DOCKER_FLYWAY_IMAGE_NAME:latest ${DOCKERHUB_ORG}/${DOCKER_FLYWAY_IMAGE_NAME}:${LATEST_TAG}
+    docker push ${DOCKERHUB_ORG}/${DOCKER_FLYWAY_IMAGE_NAME}:${LATEST_TAG}
+  else
+    echo "The 'flyway' directory does not exists in this repository."
+  fi
+}
+
+cloneHelmTemplate() {
+  GITHUB_WORKSPACE=$(pwd)
+  CHART_GITHUB_LOCATION=$GITHUB_WORKSPACE
+  TEMPLATE_REPO_GITHUB_LOCATION=$GITHUB_WORKSPACE/../convoy-helm-template
+  echo "$CHART_GITHUB_LOCATION"
+  echo "$TEMPLATE_REPO_GITHUB_LOCATION"
+  echo "TEMPLATE_REPO_GITHUB_LOCATION=$TEMPLATE_REPO_GITHUB_LOCATION"
+  echo "Cloning convoy-helm-template"
+  rm -rf $TEMPLATE_REPO_GITHUB_LOCATION
+  template_version=$(yq --exit-status '.convoy-helm-template' Chart.yaml)
+  if [ $? -eq 0 ]; then
+      echo "convoy-helm-template version: $template_version"
+  else
+      echo "convoy-helm-template version not found.Exiting."
+      exit 1
+  fi
+  git clone --branch $template_version https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com/${OWNER}/convoy-helm-template.git $TEMPLATE_REPO_GITHUB_LOCATION
+  echo "Copying the service values file"
+  cp -r $CHART_GITHUB_LOCATION/services/* $TEMPLATE_REPO_GITHUB_LOCATION/helm-chart/values/apps/
+  cp -r $CHART_GITHUB_LOCATION/Chart.yaml $TEMPLATE_REPO_GITHUB_LOCATION/helm-chart/Chart.yaml
+  set -e
+  cd $TEMPLATE_REPO_GITHUB_LOCATION/scripts
+  ls -lrta ../helm-chart/values/apps/
+}
+
+configureJava() {
+  export JAVA_HOME="/usr/lib/jvm/java-17-openjdk-amd64"
+  export PATH="/usr/lib/jvm/java-17-openjdk-amd64/bin:$PATH"
+}
+
+createPipConf() {
+  echo "[global]" > pip.conf
+  echo "index =  ${NEXUS_PROTOCOL}://${NEXUS_USERNAME}:${NEXUS_PASSWORD}@${NEXUS_SERVER_ENDPOINT}/repository/${NEXUS_PULL_REPOS_PY}/simple" >> pip.conf
+  echo "index-url = ${NEXUS_PROTOCOL}://${NEXUS_USERNAME}:${NEXUS_PASSWORD}@${NEXUS_SERVER_ENDPOINT}/repository/${NEXUS_PULL_REPOS_PY}/simple" >> pip.conf
+  echo "extra-index-url = https://pypi.org/simple" >> pip.conf
+  sudo cp pip.conf /etc/pip.conf
+  cat /etc/pip.conf
+}
+
+configureNpmrc(){
+  set +e 
+  REPO="$1"
+  requiredInputs="1"
+  validateInputs "$requiredInputs" "$@" 
+  rm ~/.npmrc
+  TOKEN=$(echo -n "${NEXUS_USERNAME}:${NEXUS_PASSWORD}" | base64 -w 0)
+  echo "//${NEXUS_SERVER_ENDPOINT}/repository/:_auth = $TOKEN" >> ~/.npmrc
+  echo "//${NEXUS_SERVER_ENDPOINT}/repository/:always-auth = true" >> ~/.npmrc 
+  echo "@tessell:registry=${NEXUS_PROTOCOL}://${NEXUS_SERVER_ENDPOINT}/repository/$REPO" >> ~/.npmrc
+  cat ~/.npmrc
+}
+
+convoyPythonBuild() {
```

Comment:
We don't have `tessell` prefix for build functions for tessell. So for build functions used in convoy also, not required.